rootfs/standard/usr/share/mynode_apps/datum/datum.json

```diff
@@ -0,0 +1,66 @@
+{
+    "name": "DATUM",
+    "short_name": "datum",
+    "author": {
+        "name": "OCEAN",
+        "link": "https://ocean.xyz"
+    },
+    "website": {
+        "name": "GitHub",
+        "link": "https://github.com/OCEAN-xyz/datum_gateway"
+    },
+    "category": "bitcoin_app",
+    "short_description": "Self-sovereign Bitcoin mining",
+    "description": [
+        "DATUM (Decentralized Alternative Templates for Universal Mining) is a brand new bitcoin mining protocol that allows you to create your own block templates using your own Bitcoin node."
+    ],
+    "latest_version": "v0.2.2",
+    "supported_archs": ["aarch64","x86_64"],
+    "download_skip": true,
+    "download_type": "source",
+    "download_source_url": "https://github.com/OCEAN-xyz/datum_gateway/archive/refs/tags/{VERSION}beta.tar.gz",
+    "download_binary_url": {
+        "aarch64": "",
+        "x86_64": ""
+    },
+    "install_env_vars": {},
+    "supports_app_page": true,
+    "supports_testnet": false,
+    "http_port": 21000,
+    "https_port": 21001,
+    "requires_bitcoin": true,
+    "requires_docker_image_installation": false,
+    "requires_electrs": false,
+    "requires_lightning": false,
+    "show_on_application_page": true,
+    "show_on_homepage": true,
+    "show_on_status_page": true,
+    "hide_status_icon": false,
+    "app_tile_name": "Datum",
+    "app_tile_running_status_text": "Running",
+    "app_tile_button_text": "Info",
+    "app_tile_button_href": "/app/datum/info",
+    "app_page_show_open_button": true,
+    "app_page_content": [
+        {
+            "heading": "Instructions",
+            "content": [
+                "If you want to mine with OEAN then enter these informations into your miners:",
+                "Stratum URL: <Your node IP>:23334",
+                "User: <Your Bitcoin address>",
+                "Password: x",
+                "",
+                "If you want to solo mine, please edit datum conf at /opt/mynode/datum/datum_config.json"
+            ]
+        }
+    ],
+    "can_uninstall": true,
+    "can_reinstall": false,
```

Comment:
I would set can_reinstall to true to help resolve any install issue faster. It just does an uninstall and install.